src/main/resources/application.properties

```diff
@@ -10,6 +10,7 @@ spring.resources.cache.cachecontrol.must-revalidate=true
 spring.resources.static-locations=classpath:/api/
 db.migrate.dropAllOnStart=false
 db.migrate.updateAllOnStart=true
+db.migrate.dataProjectNoDropAll=broad-jade-dev-data,terra-datarepo-alpha-data,terra-datarepo-staging-data
```

Comment:
Are we sure that we want to base this on the dataproject? what about int where we don't want to drop all if it's int-4 and int-5? I guess in that case dropAllOnStart is set to false?